convert2rhel/logger.py

```diff
@@ -113,15 +115,18 @@ def archive_old_logger_files(log_name, log_dir):
 
     stat = os.stat(current_log_file)
 
-    # Get the last time the file was modified
-    last_modified_at = stat.st_mtime
+    # Convert it to a local time
+    last_modified_at = localtime(int(stat.st_mtime))
+
+    # Format time to a human-readable format
+    formatted_time = strftime("%Y%m%dT%H%M%S%Z", last_modified_at)
```

Comment:
```suggestion
    # Get the last modified time in UTC
    last_modified_at = time.gmtime(stat.st_mtime)

    # Format time to a human-readable format
    formatted_time = strftime("%Y%m%dT%H%M%SZ", last_modified_at)
```

 Looking at the https://en.wikipedia.org/wiki/ISO_8601 the above format complies.